views/account.handlebars

```diff
@@ -0,0 +1,482 @@
+  <div class="page home-page">
+    <!--*********************************************************************************************************-->
+    <!--************ HERO ***************************************************************************************-->
+    <!--*********************************************************************************************************-->
+    <header class="hero">
+      <div class="hero-wrapper">
+        <!--============ Secondary Navigation ===============================================================-->
+          <div class="secondary-navigation">
```

Comment:
we should make this nav into a partial, as we have a similar nav on the index.handlebars page. we can start including the nav.handlebars instead of uploading stuff we're going to turn into partials, then removing them from everything. this one would be a good one to turn into a partial, then we can remove the index.handlebars navigation and include the partial on it.